crm_claim_rma/account_invoice.py

```diff
@@ -20,40 +21,39 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-from openerp.osv import fields, orm
-from tools.translate import _
 
+from openerp.models import Model, api, _
+from openerp.fields import Many2one
```

Comment:
Instead this limited scope, we use to import `from openerp import fields`, and then declare fields with `fields.Many2one` so that if you need to add another field type in the future, don't need to change the import